sections/translated/pl/introduction.tex

```diff
@@ -5,17 +5,18 @@
 \bigbreak
 
 \hypertarget{Heroes of Might and Magic III}{\textbf{Heroes of Might and
-Magic III: The Board Game}} is a tactical strategy RPG board game for 1-3
-players using the core box set.  The continent of Antagarich is at war as
-several different Factions, led by their Heroes, battle for
-supremacy. Choose your Faction and Hero and banish your unruly enemies from
-these lands!
+Magic III: The Board Game}} jest taktyczno-strategiczną grą planszową, która
+w wersji podstawowej pozwala na rozgrywkę w gronie 1-3 osób.  w przypadku
+gry w wersji Core. Kontynent Antagarich jest w stanie wojny, jako że kilka
+róznych Frakcji dowodzonych przez ich Bohaterów, wojują o dominację. Wybierz
+swoją Frakcję oraz Bohatera i wygnaj swoich nieprawowitych Przeciwników z
+tych ziem!
 
 \begin{multicols}{2}
-\textbf{Notice}: In this rule book, game and component terms are Capitalized.
-\textbf{Bold text} is used to draw attention to important rules.
-\textit{Italicization} is used for gameplay examples.
-\hyperlink{Heroes of Might and Magic III}{Brown colored hyperlinks} will take you to other parts of the rule book.
+\textbf{Notice}: W niniejszej księdze zasad, gra i terminy jej komponentów pisane są Wielką Literą.
```

Comment:
To właśnie problem z duplikacją :/ Czemu tu jest "Notice" zamiast "Uwaga"?

Reply:
ok już wszystko jasne, w pliku translacji jest "Uwaga", natomiast nie odpaliłem po4a, fixed